sdk/communication/communication-phone-numbers/test/lro.update.aad.spec.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Food for thought:
If we strongly believe that we should cover each test case both with connection string and AAD credential, we can export the test suite and then run it with both clients for each credential instead of duplicating the test code.

Alternatively, we pick one main credential mechanism (the connection string) and only test 1-3 small cases for DefaultAzureCredential because we have strong reason to believe that if one request succeeds to auth, the other ones will too.

Reply:
[Here's a pattern](https://github.com/Azure/azure-sdk-for-js/blob/d0760dee1e46ecc6c78d4fea16fe645e4ca6c629/sdk/formrecognizer/ai-form-recognizer/test/public/node/formrecognizerclient.spec.ts#L34) that's used by AI form recognizer to test both connection string (well, keycredential) and AAD credential. It uses a factory method to return the client and a matrix utility class

Reply:
I think it's best to cover each test case with both auth methods because we can't predict which will fail. The last time it was only getPhoneNumber that failed. I'll merge this then work on deduping test code in a new PR.